core/generics/kvstore/typedstore.go

```diff
@@ -0,0 +1,49 @@
+package kvstore
+
+import (
+	"github.com/iotaledger/hive.go/core/kvstore"
+	"github.com/iotaledger/hive.go/core/serix"
+)
+
+type DeAndSerializable interface {
+	serix.Serializable
+	serix.Deserializable
+}
+
+type TypedStored[K, V DeAndSerializable] struct {
+	kv kvstore.KVStore
+}
+
+func (t *TypedStored[K, V]) Get(key K) (value V, exists bool, err error) {
```

Comment:
This is mixing up 2 go styles. You should use a notfound error here, although not so pretty.